### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by mvktsk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+The script creates a solution template for the VirtoCommerce platform new module. The script automatically creates all the necessary projects:
+
+* Core - Business and Application Models
+* Data - Data Access and Implementation of business logic
+* Tests - xUnit Unit tests
+* Web - API Controllers
+
+and names the folders in accordance with the rules used in the VirtoCommerce development team.
+
+Difference between the script and [Virto Commerce 2.x Module Templates](https://marketplace.visualstudio.com/items?itemName=Virto-Commerce.VirtoCommerceModuleTemplates):
```
Comment (eMazeika) [inline README.md]: Differences between this script

File: README.md [hunk 2]
```diff
@@ -1,2 +1,52 @@
-# vc-powershell-template
-Powershell script for creating Virto Commerce module solution
+# VC module template
+
+## Overview
+
+This script is a lite version of [Virto Commerce 2.x Module Templates](https://marketplace.visualstudio.com/items?itemName=Virto-Commerce.VirtoCommerceModuleTemplates)
+The script creates a solution template for the VirtoCommerce platform new module. The script automatically creates all the necessary projects:
```
Comment (eMazeika) [inline README.md]: It generates a new Visual Studio solution for a module in Virto Commerce Platform.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,52 @@
...
+
+* Core - Business and Application Models
+* Data - Data Access and Implementation of business logic
+* Tests - xUnit Unit tests
+* Web - API Controllers
+
+and names the folders in accordance with the rules used in the VirtoCommerce development team.
```
Comment (eMazeika) [inline README.md]: used by the VirtoCommerce development team.

**Review by eMazeika — CHANGES_REQUESTED** [review]
- data project: EntityFramework package missing
- remove "Heineken" 
- Empty folders missing in Core, Data, Web projects. Can add?

File: README.md (hunk 1 above)
Comment (mvktsk) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (mvktsk) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (mvktsk) [reply README.md]: fixed

**Comment by mvktsk** [comment]
> * data project: EntityFramework package missing
Fixed
> * remove "Heineken"
Fixed
> * Empty folders missing in Core, Data, Web projects. Can add?
Github does not save empty folders

**Review by eMazeika — APPROVED** [review]

**Pull request merged**